src/plugins/auto/common.hpp

```diff
@@ -137,6 +137,7 @@ class AutoScheduleContext : public MultiScheduleContext {
     std::string                 _performanceHint;
     std::mutex                  _confMutex;
     MultiDeviceInferencePlugin* _plugin;
+    bool                        _isCTPUTSingleDev = false;
```

Comment:
we don't need to add the new flag to context, context is the shared flags among plugin/schedule/executable network, looks you only need to use this flag once in your logic.

Reply:
Updated, remove the new flag to context.